lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
       end
 
       # OPTIONAL - This is the detail or message that is rendered in JSON response
       # Not providing detail will render a detail the same as title, 'Operation failed'
-      # NOTE: in the future, detail will only work via i18n, not the value from response_values
       def detail
-        i18n_data[:detail].presence || response_values[:detail]
+        i18n_data[:detail] || service_response[:detail]
       end
 
       # OPTIONAL - This should usually be a developer message of some sort from the backend service
       # if one is not provided by the backend this can be nil and the key will not be rendered
       def source
-        response_values[:source]
+        service_response[:source]
```

Comment:
this is an example of one that will almost always be provided when the error is invoked, so I'm pretty satisfied with this as it is.